client/my-sites/stats/features/modules/stats-utm/stats-module-utm-dropdown.scss

```diff
@@ -65,7 +65,7 @@ $stats-utm-border: 1px solid var(--gray-gray-5, #dcdcde);
 		flex: 1; // take full width
 	}
 
-	&.is-grouped + &.is-not-grouped {
+	&.is-not-grouped + &.is-grouped {
```

Comment:
I am not quite sure if this `margin-top` is necessary as there is [a basic one](https://github.com/Automattic/wp-calypso/blob/70ba1b39504440d5d7ba3a2fd9cc17de91921dcf/client/my-sites/stats/features/modules/stats-utm/stats-module-utm-dropdown.scss#L50) for that case.

On the other hand, is that one I mentioned above a potential refactor to `&:not(:first-child)`?

Reply:
Yeah, I think you're right. The margins are still correctly applied with this rule removed.